kytos/core/tag_ranges.py

```diff
@@ -105,23 +106,35 @@ def range_intersection(
             get_validated_tags() for also list[int]
     """
     a_i, b_i = 0, 0
-    while a_i < len(ranges_a) and b_i < len(ranges_b):
+    common_diff = 1
```

Comment:
Can we give `common_diff` a more explicit meaningful name? since it's used by both `a_i` (a index) and `b_i` (b index), can we call it `index_diff` or something else similar? `common_diff` is understandable too, but when I first read I had no idea what to expect what it'd be for.

Reply:
Changed the name to `index_diff`